README.md

```diff
@@ -111,6 +111,23 @@ For example, assuming the following directory bucket name `my-test-bucket--usw2-
 usw2-az1, then the URI used will look like: `s3://my-test-bucket--usw2-az1--x-s3/<PREFIX>` (**please note that the 
 prefix for Amazon S3 Express One Zone should end with '/'**), paired with region us-west-2.
 
+## Reduce Checkpoint Loss Risks with S3 Versioning
```

Comment:
I'm not sure if I like the wording of this, but I can't think of anything better off the top of my head

Reply:
Changed title